### ISSUE TITLE
Could not authenticate with HSReplay.net

### ISSUE THREAD

**Issue by sNAtttu** [HearthSim/Hearthstone-Deck-Tracker#3858]
### Expected Behavior

I'm trying to login to HSReplay.net through the application so that I can upload my collection data.
### Actual Behavior

I'm unable to login and I get this red box with error message 

"Could not authenticate with HSReplay.net"
"Please try running HDT as administrator (right-click the exe and select 'Run as administrator').
If that does not help please try again later."

### Steps to reproduce behavior

Everytime I restart application this happens. It doesn't matter if I'm an administrator or not.
### Log/Screenshots

**Log data from hdt_log**
```
9:24:02 PM|Info|Core.Initialize >> HDT: 1.7.16.3795, Operating System: Windows 10 Enterprise 17134, .NET Framework: 461808
9:24:02 PM|Info|Updater.StartupUpdateCheck >> Checking for updates
9:24:02 PM|Info|Updater.GetReleaseUrl >> Downloading releases file
9:24:02 PM|Info|Updater.GetReleaseUrl >> using 'live' release: https://github.com/HearthSim/HDT-Releases
9:24:04 PM|Info|Updater.SquirrelUpdate >> Checking for updates (ignoreDelta=False)
9:24:05 PM|Info|Updater.SquirrelUpdate >> No new updated available
9:24:05 PM|Info|SplashScreenWindow.ShowConditional >> Showing splashscreen...
9:24:05 PM|Info|ThemeManager.LoadThemes >> Found theme: classic
9:24:05 PM|Info|ThemeManager.LoadThemes >> Found theme: dark
9:24:05 PM|Info|ThemeManager.LoadThemes >> Found theme: frost
9:24:05 PM|Info|ThemeManager.LoadThemes >> Found theme: minimal
9:24:05 PM|Info|GameV2.Reset >> -------- Reset ---------
9:24:07 PM|Info|OverlayWindow.SetTopmost >> Hearthstone window not found
9:24:08 PM|Info|GameV2.Reset >> -------- Reset ---------
9:24:08 PM|Info|BackupManager.Run >> Running BackupManager
9:24:08 PM|Info|BackupManager.Run >> Backup for today already exists
9:24:08 PM|Info|PluginManager.LoadPluginsFromPath >> Loading Plugins...
9:24:08 PM|Info|LogWatcherManager.Start >> Using Hearthstone log directory 'D:\Battle.net\Hearthstone\Logs'
9:24:08 PM|Info|ApiWrapper.UpdateUploadTokenStatus >> Checking token status...
9:24:08 PM|Info|HsReplayDecks.Load >> Loaded from disk: Count=2629, ServerTS=1/11/2019 7:10:15 PM, Downloaded=1/11/2019 9:23:26 PM Age=00:00:42.2257475
9:24:08 PM|Info|HsReplayDecks.Load >> Complete: Count=2629, ServerTS=1/11/2019 7:10:15 PM, Downloaded=1/11/2019 9:23:26 PM Age=00:00:42.2257475
9:24:08 PM|Info|ApiWrapper.UpdateUploadTokenStatus >> Token is not claimed
9:24:10 PM|Info|HsReplayDeckInfoView.OpenTrendingDecksCommand >> [Helper.TryOpenUrl] https://hsreplay.net/decks/trending/?utm_source=hdt&utm_medium=client&utm_campaign=trending
9:24:25 PM|Info|HSReplayNetOAuth.Authenticate >> Authenticating with HSReplay.net...
9:24:25 PM|Error|HSReplayNetOAuth.Authenticate >> System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.Net.HttpListener..ctor()
   at HSReplay.OAuth.CallbackListener.Create(Int32[] ports) in C:\projects\hearthstone-deck-tracker\HSReplay-Api\HSReplay\OAuth\CallbackListener.cs:line 57
   at HSReplay.OAuth.OAuthClient.GetAuthenticationUrl(Scope[] scopes, Int32[] ports) in C:\projects\hearthstone-deck-tracker\HSReplay-Api\HSReplay\OAuth\OAuthClient.cs:line 106
   at Hearthstone_Deck_Tracker.HsReplay.HSReplayNetOAuth.<Authenticate>d__30.MoveNext() in C:\projects\hearthstone-deck-tracker\Hearthstone Deck Tracker\HsReplay\HSReplayNetOAuth.cs:line 58
9:24:25 PM|Info|ErrorManager.AddError >> New error: Could not authenticate with HSReplay.net
Please try running HDT as administrator (right-click the exe and select 'Run as administrator').
If that does not help please try again later.
9:25:30 PM|Info|HSReplayNetOAuth.Authenticate >> Authenticating with HSReplay.net...
9:25:30 PM|Error|HSReplayNetOAuth.Authenticate >> System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.Net.HttpListener..ctor()
   at HSReplay.OAuth.CallbackListener.Create(Int32[] ports) in C:\projects\hearthstone-deck-tracker\HSReplay-Api\HSReplay\OAuth\CallbackListener.cs:line 57
   at HSReplay.OAuth.OAuthClient.GetAuthenticationUrl(Scope[] scopes, Int32[] ports) in C:\projects\hearthstone-deck-tracker\HSReplay-Api\HSReplay\OAuth\OAuthClient.cs:line 106
   at Hearthstone_Deck_Tracker.HsReplay.HSReplayNetOAuth.<Authenticate>d__30.MoveNext() in C:\projects\hearthstone-deck-tracker\Hearthstone Deck Tracker\HsReplay\HSReplayNetOAuth.cs:line 58
9:25:30 PM|Info|ErrorManager.AddError >> New error: Could not authenticate with HSReplay.net
Please try running HDT as administrator (right-click the exe and select 'Run as administrator').
If that does not help please try again later.
9:26:04 PM|Info|HSReplayNetOAuth.Authenticate >> Authenticating with HSReplay.net...
9:26:04 PM|Error|HSReplayNetOAuth.Authenticate >> System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.Net.HttpListener..ctor()
   at HSReplay.OAuth.CallbackListener.Create(Int32[] ports) in C:\projects\hearthstone-deck-tracker\HSReplay-Api\HSReplay\OAuth\CallbackListener.cs:line 57
   at HSReplay.OAuth.OAuthClient.GetAuthenticationUrl(Scope[] scopes, Int32[] ports) in C:\projects\hearthstone-deck-tracker\HSReplay-Api\HSReplay\OAuth\OAuthClient.cs:line 106
   at Hearthstone_Deck_Tracker.HsReplay.HSReplayNetOAuth.<Authenticate>d__30.MoveNext() in C:\projects\hearthstone-deck-tracker\Hearthstone Deck Tracker\HsReplay\HSReplayNetOAuth.cs:line 58
9:26:36 PM|Info|HsReplayDeckInfoView.OpenTrendingDecksCommand >> [Helper.TryOpenUrl] https://hsreplay.net/decks/trending/?utm_source=hdt&utm_medium=client&utm_campaign=trending
9:26:41 PM|Info|HSReplayNetOAuth.Authenticate >> Authenticating with HSReplay.net...
9:26:41 PM|Error|HSReplayNetOAuth.Authenticate >> System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.Net.HttpListener..ctor()
   at HSReplay.OAuth.CallbackListener.Create(Int32[] ports) in C:\projects\hearthstone-deck-tracker\HSReplay-Api\HSReplay\OAuth\CallbackListener.cs:line 57
   at HSReplay.OAuth.OAuthClient.GetAuthenticationUrl(Scope[] scopes, Int32[] ports) in C:\projects\hearthstone-deck-tracker\HSReplay-Api\HSReplay\OAuth\OAuthClient.cs:line 106
   at Hearthstone_Deck_Tracker.HsReplay.HSReplayNetOAuth.<Authenticate>d__30.MoveNext() in C:\projects\hearthstone-deck-tracker\Hearthstone Deck Tracker\HsReplay\HSReplayNetOAuth.cs:line 58
9:26:42 PM|Info|HSReplayNetOAuth.Authenticate >> Authenticating with HSReplay.net...
9:26:42 PM|Error|HSReplayNetOAuth.Authenticate >> System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.Net.HttpListener..ctor()
   at HSReplay.OAuth.CallbackListener.Create(Int32[] ports) in C:\projects\hearthstone-deck-tracker\HSReplay-Api\HSReplay\OAuth\CallbackListener.cs:line 57
   at HSReplay.OAuth.OAuthClient.GetAuthenticationUrl(Scope[] scopes, Int32[] ports) in C:\projects\hearthstone-deck-tracker\HSReplay-Api\HSReplay\OAuth\OAuthClient.cs:line 106
   at Hearthstone_Deck_Tracker.HsReplay.HSReplayNetOAuth.<Authenticate>d__30.MoveNext() in C:\projects\hearthstone-deck-tracker\Hearthstone Deck Tracker\HsReplay\HSReplayNetOAuth.cs:line 58

```

**Comment by riQQ** [comment]
Does entering the following commands in the command line help?
```
sc config http start=demand
sc start http
```
(see https://stackoverflow.com/questions/45899696/why-i-cant-use-httplistener-in-my-windows-10-computer-system-platformnotsuppo)

**Comment by sNAtttu** [comment]
Thank you for the link! I actually tried to run those commands and the command "sc start http" gave me this
```
System error 1009 has occurred.

The configuration registry database is corrupt.
```
After doing some investigation on the topic I found that one of the sub-keys was missing values in 

```
HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\HTTP\Parameters\SslBindingInfo
```
After deleting that sub-key I was able to start the http service. When the service was up and running I was able to authenticate to the HSReplay.net 👍 

Just out of curiosity how did you find the stack overflow link? I tried to find something also on the topic before creating this issue but I was unable to find that one.
